04-onion-routing.md

```diff
@@ -536,16 +533,22 @@ outgoing channel:
 1. type: UPDATE|13 (`incorrect_cltv_expiry`)
 2. data:
    * [`4`:`cltv_expiry`]
-   * [`2`:`len`]
-   * [`len`:`channel_update`]
+   * [`92`:`channel_update`]
```

Comment:
Hmm, by moving to a fixed size message here, aren't we locking ourselves in a bit by assuming the `channel_update` message will _always_ be 92 bytes?